data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3057,6 +3057,50 @@ spec:
                       use for instances in this cluster. Deprecated: use FlavorName
                       in DefaultMachinePlatform to define default flavor.'
                     type: string
+                  controlPlanePort:
+                    description: ControlPlanePort contains the control plane port
+                      details, with details of one network containing two openstack
+                      subnets for dualstack clusters or one IPv4 openstack subnet.
+                      These network and subnets will be used by all nodes created
+                      by the installer. By setting this, the installer will no longer
+                      create a network and subnet. The subnets and network specified
+                      in ControlPlanePort will not be deleted or modified by the installer.
+                      This is a TechPreview feature and requires setting featureSet
+                      to TechPreviewNoUpgrade.
+                    properties:
+                      fixedIPs:
+                        description: Specify subnets of the network where control
+                          plane port will be discovered.
+                        items:
+                          description: FixedIP identifies a subnet defined by a subnet
+                            filter.
+                          properties:
+                            subnet:
+                              description: SubnetFilter defines a subnet by ID and/or
+                                name.
+                              properties:
+                                id:
+                                  type: string
+                                name:
+                                  type: string
+                              type: object
+                          required:
+                          - subnet
+                          type: object
+                        type: array
+                      network:
+                        description: Network is a query for an openstack network that
+                          the port will be discovered on. This will fail if the query
+                          returns more than one network.
+                        properties:
+                          id:
+                            type: string
+                          name:
+                            type: string
```

Comment:
Is the network required here?

Reply:
No, because `machinesSubnet` only contains info about the subnet and that would be the only info added to `controlPlanePort` during conversion.